pyaedt/modules/Material.py

```diff
@@ -26,143 +27,111 @@ class MatProperties(object):
     defaultvalue = [1.0,             1.0,            0,              0,                         0,                       0.01,                      0,              0,               0,                               0,                  0,                  0.8,         0,                   0,                 0,                   0,                      0,                          0]
     defaultunit  = [None,            None,          '[siemens m^-1]', None,                     None,                   '[W m^-1 C^-1]',        '[Kg m^-3]',    '[J Kg^-1 C^-1]', '[C^-1]',                       '[Pa]',             None,               None,               None,               None,               None,           None,                  None,                       None]
     diel_order =   [3, 0, 1, 4, 5, 6, 7, 8, 9, 10, 11, 1]
     cond_order =   [2, 0, 1, 4, 5, 6, 7, 8, 9, 10, 11, 3]
 
     @classmethod
-    def get_defaultunit(cls, fullname=None, catname=None, aedtname=None):
+    def get_defaultunit(cls, aedtname=None):
         """Get the defaultunit for a given fullname or catname
 
         Parameters
         ----------
-        fullname :
-            optiona full name of the property (Default value = None)
-        catname :
-            optional Category name (Default value = None)
-        aedtname :
-            optional aedtname (Default value = None)
+        aedtname : str
+            aedtname (Default value = None)
 
         Returns
         -------
-        type
+        str
             defaultunit if exists
 
         """
-        if fullname:
-            return cls.defaultunit[cls.fullname.index(fullname)]
-        elif catname:
-            return cls.defaultunit[cls.catname.index(catname)]
-        elif aedtname:
+        if aedtname:
             return cls.defaultunit[cls.aedtname.index(aedtname)]
         else:
             raise TypeError("get_defaultunit: either fullname or catname MUST be defined")
 
+
     @classmethod
-    def get_aedtname(cls, fullname=None, catname=None):
+    def get_defaultvalue(cls, aedtname):
         """Get the defaultunit for a given fullname or catname
 
         Parameters
         ----------
-        fullname :
-            optiona full name of the property (Default value = None)
-        catname :
-            optional Category name (Default value = None)
+        aedtname : str
+             aedtname (Default value = None)
 
         Returns
         -------
-        type
-            aedtname
+        float
+            defaultunit
 
         """
-        if fullname:
-            return cls.aedtname[cls.fullname.index(fullname)]
-        elif catname:
-            return cls.aedtname[cls.catname.index(catname)]
+        if aedtname:
+            return cls.defaultvalue[cls.aedtname.index(aedtname)]
         else:
-            raise TypeError("get_aedtname: either fullname or catname MUST be defined")
+            raise TypeError("get_defaultunit: either fullname or catname MUST be defined")
```

Comment:
Let's work on not capitalizing words to tell the user something.  None of the other python libraries out there (at least not the big ones) do this, and it seems bad form.  Generally the blood red text is enough to get the user's attention.